### PR TITLE
Migrate spices

### PR DESCRIPTION
**Pull request by Cyrano7**
Migrate spices, normalized curry powder, updated seasoning salt recipe. Some spice values were changed slightly to better match with thier 100 charges vaule.

### CODE REVIEW

File: data/json/items/comestibles/spice.json [hunk 1]
```diff
@@ -0,0 +1,125 @@
+[
+  {
+    "id": "cinnamon",
+    "copy-from": "salt",
```
Comment (mugling) [inline data/json/items/comestibles/spice.json]: You should copy from an `abstract spice` as opposed to `salt`

Comment (Cyrano7) [inline data/json/items/comestibles/spice.json]: Done

**Comment by Cyrano7** [comment]
@mugling This is why I don't migrate things anymore. Can never do it quite right.

**Comment by mugling** [comment]
You did pretty well though and it wasn't that hard for me to clean the last of it up. Identifying common patterns is half of the work so if you spot any more feel free to continue opening PR's.

**Pull request merged**